ampoule/test/test_process.py

```diff
@@ -118,6 +118,18 @@ def second(self):
         return {}
     Second.responder(second)
 
+class HangForever(amp.Command):
+    pass
+
+class TimingOutChild(child.AMPChild):
+    def hang_forever(self):
+        return defer.Deferred()
+    HangForever.responder(hang_forever)
```

Comment:
Ampoule predates the use of the decorator syntax, but we can use it now :).  We can just spell this as `@HangForever.responder` above the method.

Reply:
Fixed in 6abc4db